web/crux-ui/src/components/products/versions/images/select-images-card.tsx

```diff
@@ -187,17 +218,17 @@ const SelectImagesCard = (props: SelectImagesCardProps) => {
             className="max-w-lg"
             name="imageName"
             grow
-            label={t('imageName')}
-            value={filter}
-            onChange={event => onFilterChange(event.target.value)}
+            label={registry && registry.type === 'unchecked' ? t('imageNameAndTag') : t('imageName')}
+            value={filterOrName}
+            onChange={event => onInputChange(event.target.value)}
             messageType="info"
-            message={
-              filter?.length < IMAGE_FILTER_MIN_LENGTH ? t('filterMinChars', { length: IMAGE_FILTER_MIN_LENGTH }) : null
-            }
+            message={inputMessage}
           />
 
-          {filterResult.length < 1 ? (
-            filter.length < 1 ? null : searching ? (
+          {registry && registry.type === 'unchecked' ? (
```

Comment:
is registry ever going to be null?

Reply:
Yes on the very first render, before `useEffect` sets it to something.